pkg/common/cns-lib/vsphere/utils.go

```diff
@@ -391,3 +395,35 @@ func isVsan67u3Release(ctx context.Context, m *defaultVirtualCenterManager, host
 	log.Debugf("vCenter version is :%q", vc.Client.Version)
 	return vc.Client.Version == cns.ReleaseVSAN67u3, nil
 }
+
+// IsvSphereVersion70U3orAbove checks if specified version is 7.0 Update 3 or higher
+// The method takes aboutInfo{} as input which contains details about
+// VC version, build number and so on.
+// If the version is 7.0 Update 3 or higher, the method returns true, else returns false
+// along with appropriate errors during failure cases
+func IsvSphereVersion70U3orAbove(ctx context.Context, aboutInfo types.AboutInfo) (bool, error) {
+	log := logger.GetLogger(ctx)
+	items := strings.Split(aboutInfo.ApiVersion, ".")
+	apiVersion := strings.Join(items[:], "")
+	// Convert version string to string, Ex: "7.0.3" becomes 703, "7.0.3.1" becomes 703
+	vSphereVersionInt, err := strconv.Atoi(apiVersion[0:3])
+	if err != nil {
+		msg := fmt.Sprintf("Error while converting ApiVersion %q to integer, err %+v", apiVersion, err)
+		log.Errorf(msg)
+		return false, errors.New(msg)
+	}
+	vSphere70u3VersionStr := strings.Join(strings.Split(VSphere70u3Version, "."), "")
+	vSphere70u3VersionInt, err := strconv.Atoi(vSphere70u3VersionStr[0:3])
+	if err != nil {
+		msg := fmt.Sprintf("Error while converting VSphere70u3Version %q to integer, err %+v", VSphere70u3Version, err)
```

Comment:
nit: `error ...`
this applies to the rest of the PR too

Reply:
Done